### ISSUE TITLE
version 11.03.2022

### ISSUE THREAD

**Issue by MickCasanova** [keyofrassilon/Sims4ScriptCore#2]
Got pummeled with this error.. So many I lost count. Then it stopped. Then it happened again..

[debugger.log](https://github.com/keyofrassilon/Sims4ScriptCore/files/9934411/debugger.log)
The fun begins at line 83 (I'm using Notepad++) Or if you prefer, the timestamp is "Thu Nov  3 20:46:22 2022"

**Comment by MickCasanova** [comment]
So I reloaded the mod and the only thing I changed in the config is the number of sims on the lot. I left everything else in the directory alone. So far it seems the error is related to my config file because once reloaded, the errors stopped. I'm going to see what I can work out and when I find the portion in the config that causes the issue, I will let you know.

**Comment by MickCasanova** [comment]
Well it appears that it may be the spawn.ini If I clear the data from it (simply empty the file) then the errors start. Even if I remove just "["Paige Von Winter", "Paige Harris II", "Crusty Zombie"]" Since none of these sims are in my game, the errors also return. 

Also noticed another oddity... Every sim in my house that isn't a resident, has a "Visitor" tag over their head. This needs to be optional. No real need for me to see it since I already know who is part of the household.

Also, seems that my mail person no longer has a uniform. Haven't checked other roles, but this is slightly annoying. I like role sims to dress like they do something. 
All in all, acting a little more off the wall than I like. Reverting back to 10.23.22 for now.

**Comment by keyofrassilon** [comment]
Yup spawn.ini needs to stay filled. Change sims to sims = [] to empty that field. I'll write in some error trap stuff tomorrow and fix the titles

**Comment by MickCasanova** [comment]
In the latest, I get no mixologist at all. Bar says he's supposed to be there, but he's not. Not sure about any of the other role sims. I'm guessing if one isn't working, none of them are.

**Comment by keyofrassilon** [comment]
Chances are there is no assigned mixologist if you are using the routine. Set the disable_routine to True in config and reload the config. Best to reload routine from the menu. Main > Control > Load Routine. This should load the vanilla assignemnts per lot. Otherwise pick a sim you want as a mixologist and place them in sims.ini under [career_Adult_NPC_Bartender] with first name and last name as shown already in there. If you erased everything in sims.ini, refer to the original on this site to see how they are set up.
